lms/templates/problem_notifications.html

```diff
@@ -0,0 +1,17 @@
+<%page expression_filter="h" args="notification_type, notification_icon, submission_response_message, points_earned, points_possible"/>
+<%!
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import HTML
+%>
+<%
+    notification_type = "partially-correct"
+    notification_icon = "asterisk"
+    submission_response_message = "Partially correct"
```

Comment:
We need the ability to pass in an arbitrary message, not submission_response_message, points_earned, points_possible. These are examples of messages that we will need to pass in:

1. Partially correct (3/4 points earned)
2. Your answers have been saved but not graded. Click 'Submit' to grade them.
3. Hint 2/2: A fruit has seeds
    Hint 1/2: Fruit plants flower